go/vt/discovery/topology_watcher.go

```diff
@@ -198,7 +208,7 @@ func (tw *TopologyWatcher) loadTablets() {
 	}
 
 	for alias, newVal := range newTablets {
-		if tw.tabletFilter != nil && !tw.tabletFilter.IsIncluded(newVal.tablet) {
+		if tw.tabletFilters != nil && !tw.hasTabletFiltersMatch(newVal.tablet) {
```

Comment:
Wait a second. I didn't change this, but shouldn't this be _(or condition)_:
```
if tw.tabletFilters != nil || !tw.hasTabletFiltersMatch(newVal.tablet) {
```
🤔?